.gitignore

```diff
@@ -124,3 +124,6 @@ Temporary Items
 
 # VIM
 *.swp
+
+# Visual Studio Code
+.vscode
```

Comment:
Actually it's common practice to check in editor configs. That way you can have the same settings across different local repos in different machines. Every typescript project I've seen has a .vscode folder in it.